hbw/app/services/hbw/with_definitions.rb

```diff
@@ -28,6 +32,25 @@ def process_definitions
       end
     end
 
+    def process_definitions_with_starter_candidates(user_email)
+      Rails.cache.fetch(:process_definitions) do
```

Comment:
AFAIK it will cache process definitions of the first requested user and then will always return the same value.

Reply:
Ok, fixed